src/main/java/io/khasang/jrepetitor/entity/User.java

```diff
@@ -0,0 +1,65 @@
+package io.khasang.jrepetitor.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Transient
+    private static final String ROLE = "ROLE_USER";
```

Comment:
For what?

Reply:
private static final String ROLE - for no-arg constructor